src/test/resources/fixtures/test-fixtures/textarea-with-spellcheck-enabled/output.txt

```diff
@@ -0,0 +1,6 @@
+<div class="govuk-form-group">
+  <label class="govuk-label" for="textarea-with-spellcheck-enabled">
+    Spellcheck is enabled
+  </label>
+  <textarea class="govuk-textarea" id="textarea-with-spellcheck-enabled" name="spellcheck" rows="5" spellcheck="true"></textarea>
+</div>
```

Comment:
As above for the use case where spellcheck is undefined.

Reply:
As above - there are other examples which won't be clear in the diff, for example https://github.com/hmrc/play-frontend-govuk/blob/upgrade-to-3.8.0/src/test/resources/fixtures/test-fixtures/textarea-with-default-value/input.json